donut-app/resources-fenix3/settings/properties.xml

```diff
@@ -0,0 +1,13 @@
+<properties>
+    <property id="screenWidth" type="number">218</property>
```

Comment:
I notice that all of the circle watches have this resolution. Is there a way to make these properties apply to all of those watches, too?